WalletWasabi.Tests/UnitTests/Crypto/BlindingTests.cs

```diff
@@ -0,0 +1,180 @@
+using NBitcoin;
+using NBitcoin.DataEncoders;
+using Newtonsoft.Json;
+using System;
+using System.IO;
+using System.Linq;
+using System.Numerics;
+using System.Text;
+using WalletWasabi.Crypto;
+using WalletWasabi.JsonConverters;
+using Xunit;
+using static WalletWasabi.Crypto.SchnorrBlinding;
+
+namespace WalletWasabi.Tests.UnitTests.Crypto
+{
+	public class SchnorrBlindindSignatureTests
```

Comment:
Maybe change the class name to match the file name?
If not then notice that there is a typo `Blindind`

Reply:
ACK, good catch!